src/Components/Artwork/GridItem.tsx

```diff
@@ -88,32 +102,38 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
       currentUserSpread = { currentUser }
     }
     return (
-      <div className={className} style={style}>
-        <Placeholder style={{ paddingBottom: artwork.image.placeholder }}>
-          <a href={artwork.href}>
-            <Image src={this.imageURL} innerRef={img => (this.image = img)} />
-          </a>
-          <Responsive>
```

Comment:
@alloy - Should we maybe hoist `<Responsive>` up a level into ArtworkGrid? Seems like we could pass values into `GridItem` instead and save a lot of operations.